pkg/security/probe/constantfetch/fallback.go

```diff
@@ -1004,3 +1004,14 @@ func getSockStructSKProtocolOffset(kv *kernel.Version) uint64 {
 		return ErrorSentinel
 	}
 }
+
+func getFlowiProtoOffset(kv *kernel.Version) uint64 {
+	switch {
+	case kv.IsRockyKernel() && kv.IsInRangeCloseOpen(kernel.Kernel5_14, kernel.Kernel5_15):
+		return 18
```

Comment:
Do we need this extra case statement if the returned value is the same as the returned one?

Reply:
My bad I didn't notice the overlap between the kernel version ranges